cmd/ketch/framework_remove.go

```diff
@@ -48,6 +49,9 @@ func frameworkRemove(ctx context.Context, cfg config, options frameworkRemoveOpt
 		return fmt.Errorf("failed to get framework: %w", err)
 	}
 
+	if err := pruneRemovedAppsFromStatus(ctx, cfg, framework); err != nil {
+		return fmt.Errorf("failed to update framework's apps: %w", err)
```

Comment:
Do you think it would be better to have the message say `failed to prune framework's apps:` just for a little bit more clarification?

Reply:
It sure would. Change made.